src/reducers/reducer.js

```diff
@@ -89,4 +90,14 @@ function recordHistory(reducer) {
     }
 }
 
-export default recordHistory(reducer);
+function rootReducer(reducer) {
+    return function(state, action) {
+        if (action.type === 'RELOAD') {
+            const {history} = state;
```

Comment:
I there extra stuff in `state` that is stripped when you do this? I don't see what these two lines do, I don't think `state` is changed here

Reply:
Everything but the history is stripped down, it triggers a soft reload.

Reply:
Oh okay so everything else gets set to default except the history, looks good

Reply:
Hmm, can you not combine this reducer into the `recordHistory` function that's already there? I'm not sure I understand why it is a separate reducer.

Reply:
I think it should be separate from `recordHistory`. This reducer doesn't really change the history, it just removes everything else. It could have a different name though for sake of clarity, maybe like `preserveHistoryOnReload`